### PR TITLE
Enhancement: Enable and configure global_namespace_import fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] enables and configures the `global_namespace_import` fixer
* [x] runs `make cs`
* [x] runs `make baseline`

Follows #157.

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.17.2/doc/rules/import/global_namespace_import.rst.

### CODE REVIEW

File: src/Faker/ORM/Propel/ColumnTypeGuesser.php [hunk 1]
```diff
@@ -2,9 +2,6 @@
 
 namespace Faker\ORM\Propel;
 
-use ColumnMap;
-use PropelColumnTypes;
```
Comment (localheinz) [inline src/Faker/ORM/Propel/ColumnTypeGuesser.php:6]: ❓

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
👎 waste of time

File: src/Faker/ORM/Propel/ColumnTypeGuesser.php (hunk 1 above)
Comment (pimjansen) [reply src/Faker/ORM/Propel/ColumnTypeGuesser.php:6]: Wth didnt even knew these

**Review by pimjansen — APPROVED** [review]

**Comment by localheinz** [comment]
Thank you, @pimjansen!

**Pull request merged**